runbenchmark.py

```diff
@@ -15,8 +15,8 @@
 parser.add_argument('framework', type=str,
                     help="The framework to evaluate as defined by default in resources/frameworks.yaml.")
 parser.add_argument('benchmark', type=str, nargs='?', default='test',
-                    help="The benchmark type to run as defined by default in resources/benchmarks/{benchmark}.yaml "
-                         "or the path to a benchmark description file. Defaults to `%(default)s`.")
+                    help="The benchmark type to run as defined by default in resources/benchmarks/{benchmark}.yaml, "
+                         "a path to a benchmark description file, or an openml suite ID. Defaults to `%(default)s`.")
```

Comment:
suggestion what about using sth similar to the nifty openml urls:
```bash
# to run the entire openml suite:
python runbenchmark.py gama openml/s/42 

# to run a single openml task (without the need for a yaml definition)
python runbenchmark.py gama openml/t/42 
```
this leaves this parameter open for future extensions, for example to provide easy support for kaggle datasets:
```
python runbenchmark.py gama kaggle/some/dataset
```

Reply:
ideally, in future versions, for consistency we could do the same for the benchmark definitions.
Instead of 
```
- name: Airlines
  openml_task_id: 189354
```
have
```
- name: Airlines
   dataset: openml/t/189354

- name: CreditCardFraud
   dataset: kaggle/mlg-ulb/creditcardfraud
```
to be aligned with current support for external files, cf. https://github.com/openml/automlbenchmark/blob/master/examples/custom/benchmarks/filedatasets.yaml

Reply:
Just pushed some changes that works towards this refactoring. It's a bit slower than I'm used to since I'm still getting the hang of the infrastructure.

Reply:
no one here is in a hurry, are you?
As you started to refactor this, could you please move your new files into a submodule, in a way similar to the `datasets` submodule. I'd suggest sth like `definitions` or `argsparser`,  exposing only your `benchmark_load` for now.

Reply:
> Should I also overload the task specification parameter -t to directly run on a task? (Obviously both the benchmark framework and AutoML system must support the task type for a successful run)

Just noticed this comment.
If you follow my previous suggestion, you shouldn't to override this because `-t` doesn't make any sense when calling:
```
python runbenchmark.py gama openml/t/42 
```

If you were asking if we should allow
```
python runbenchmark.py gama openml/s/226 -t helena
```
I think it's necessary anyway for parallelism.
User triggers the entire suite with `python runbenchmark.py gama openml/s/226`, but each ec2 instance for example will run only something like `python runbenchmark.py gama openml/s/226 -t helena -f 3`.

Reply:
Both of these options work already (automatically 👍 )